src/index.md

```diff
@@ -1,37 +1,142 @@
 ---
 layout: default
----
-
-# Apache Beam
+body_class: body--index
 
-#### Apache Beam provides an advanced unified programming model, allowing you to implement batch and streaming data processing jobs that can run on any execution engine.
+logos:
+- title: APEX
+  image_url: /images/logo_apex.png
+  url: "http://apex.apache.org"
+- title: Flink
+  image_url: /images/logo_flink.png
+  url: "http://flink.apache.org"
+- title: Spark
+  image_url: /images/logo_spark.png
+  url: http://spark.apache.org/
+- title: Google Cloud Dataflow
+  image_url: /images/logo_google_cloud.png
+  url: https://cloud.google.com/dataflow/
 
-Apache Beam is:
-* **UNIFIED** - Use a single programming model for both batch and streaming use cases.
-* **PORTABLE** - Execute pipelines on multiple execution environments, including Apache Apex, Apache Flink, Apache Spark, and Google Cloud Dataflow.
-* **EXTENSIBLE** - Write and share new SDKs, IO connectors, and transformation libraries. 
+pillars:
+- title: Unified
+  body: Use a single programming model for both batch and streaming use cases.
+- title: Portable
+  body: Execute pipelines on multiple execution environments.
+- title: Extensible
+  body: Write and share new SDKs, IO connectors, and transformation libraries.
 
-## Get Started
-
-To use Beam for your data processing tasks, start by reading the [Beam Overview]({{ site.baseurl }}/get-started/beam-overview) and performing the steps in the [Quickstart]({{ site.baseurl }}/get-started/quickstart). Then dive into the [Documentation]({{ site.baseurl }}/documentation/) section for in-depth concepts and reference materials for the Beam model, SDKs, and runners.                                    
+cards:
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 1
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 2
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 3
+---
+<div class="hero section">
+  <div class="hero__cols">
+    <div class="hero__cols__col">
+      <div class="hero__cols__col__content">
+        <div class="hero__title">
+          Apache Beam: An advanced unified programming model
+        </div>
+        <div class="hero__subtitle">
+          Implement batch and streaming data processing jobs that run on any execution engine.
+        </div>
+        <div class="hero__ctas">
+          <a class="button button--primary" href="/get-started/beam-overview/">Learn more</a>
+          <a class="button" href="/get-started/quickstart-java/">Quickstart</a>
```

Comment:
Yeah - something i'd like to do. In the meantime, would you be OK with buttons like "Java Quickstart" and "Python Quickstart"?

In the long-run, it'd be nice to have a canonical quickstart page that we can link to - with a general intro and links to either Python or Java.